models/users.js

```diff
@@ -0,0 +1,28 @@
+import { DataTypes } from 'sequelize';
+import sequelize from '../config/db.js';
+
+const User = sequelize.define('User', {
+  id: {
+    type: DataTypes.UUID,
+    defaultValue: DataTypes.UUIDV4,
+    allowNull: false,
+    primaryKey: true,
+  },
+  email: {
+    type: DataTypes.STRING,
+    allowNull: false,
+    unique: true,
+  },
+  fullname: {
+    type: DataTypes.STRING,
+    allowNull: false,
```

Comment:
can we add validation here? Name should be at least two words

Reply:
"Yup" is being used to validate the data.